### DISCUSSION TITLE
Error on Notification component due to missing Icon

### DISCUSSION THREAD

**Discussion by blockhunts** [wireui/wireui#620] [BUG]
I have put a notification component inside my <body> and triggered through this code
`$this->notification()->success(
            $title = 'Profile saved',
            $description = 'Your profile was successfully saved'
        );`
      
 But unfortunately I got this error
![image](https://user-images.githubusercontent.com/38546608/219253632-59248b14-f3a3-4f0b-a0b0-c9ac5f57ea5a.png)
![image](https://user-images.githubusercontent.com/38546608/219253752-ce8839ae-f727-4211-af44-22f6b5ff0d1a.png)

I already guessed the reason for this problem is because the icon path is not valid. And then after I changed the URL to local custom domain, the component work as intended.

Failed Path:
`http://192.168.0.123/project_name/public/page_name`

Success Path:
`http://project_name.test/page_name`

But I want other computer to be able to access my program localy. Hence using local custom domain is not reccomended.
Is there anyway to configure the path, so that it will find the icon path correctly?

**Comment by Raised** [comment]
How did it go? Did you find out a solution? I get the same error when I deploy my app to a shared hosting and the app is not in root directory. All other icons and functions of wireui seems to work as expected but not notification. It looks exactly as it does for you.

**Reply by blockhunts** [reply]
nope, I tried to dwell into the code but I cannot find the fix. So I gave up using this component

**Reply by Raised** [reply]
I will give it one more try, it would looks so nice. Thanks for your reply 😊

**Reply by insulae** [reply]
i have a "ugly" fix for this:

https://github.com/wireui/wireui/discussions/703
